content/en/docs/about/contributing.md

```diff
@@ -103,13 +106,34 @@ A typical OWNERS file looks like:
 ```yaml
 approvers:
   - alice
-  - bob     # this is a comment
+  - bob # this is a comment
 reviewers:
   - alice
-  - carol   # this is another comment
+  - carol # this is another comment
   - sig-foo # this is an alias
 ```
 
+#### Emeritus
+
+It is inevitable, but there are times when someone may shift focuses, change jobs or step away from
+a specific area in the project for a time. These people may be domain experts over certain areas of
+the codebase, but can no longer dedicate the time needed to handle the responsibilities of
+reviewing and approving changes. They are encouraged to add themselves as an “emeritus”
+approver under the `emeritus_approvers` key.
+
+GitHub usernames listed under the `emeritus_approvers` key can no longer approve code
+(use the /approve command) and will be ignored by prow for assignment. However, it can still be
+referenced by a person looking at the OWNERS file for a possible second or more informed opinion.
+
+When a contributor returns to being more active in that area, they may be promoted back to a
+regular approver at the discretion of the current approvers.
+
+```yaml
+emeritus_approvers:
+  - david # 2018-05-02
```

Comment:
Is this start or end time?

Reply:
I think, it is end term.
However, it doesn't look like Kubernetes adds it:
- https://github.com/kubernetes/kubernetes/blob/master/OWNERS#L11
- https://github.com/kubernetes-sigs/kueue/blob/main/OWNERS#L15

Reply:
I think it's inconsistently applied. I've seen it in some projects but not all.

Reply:
We can probably remove this info then. People can track it in commit history anyways.